### ISSUE TITLE
1.30.0-beta.6 failing on `createType(ExtrinsicSignatureV4)` with some older Kusama blocks

### ISSUE THREAD

**Issue by emostov** [polkadot-js/api#2508]
# Error

I went to go bump substrate-api-sidecar's @polkadot/api from 1.29.1 to 1.30.0-beta.6 and started encountering errors starting with `createType(SignedBlock)` on older Kusama blocks that do not occur when I go back to 1.29.1.

`localhost:8080/block/900` -> `createType(SignedBlock):: Struct: failed on 'block':: Struct: failed on 'extrinsics':: createType(ExtrinsicV4):: createType(ExtrinsicSignatureV4):: Maximum call stack size exceeded`

And in the terminal Sidecar logs: `2020-08-23 16:16:32        RPC-CORE: getBlock(hash?: BlockHash): SignedBlock:: createType(SignedBlock):: Struct: failed on 'block':: Struct: failed on 'extrinsics':: createType(ExtrinsicV4):: createType(ExtrinsicSignatureV4):: Maximum call stack size exceeded`.

I think the error originates from this call to `getBlock`: https://github.com/paritytech/substrate-api-sidecar/blob/master/src/services/blocks/BlocksService.ts#L42

# How to produce error

To recreate the error (you will have to install `wasm-pack` if you don't already have it):
```
git clone https://github.com/paritytech/substrate-api-sidecar.git
git checkout zeke-1-3-b # commit e0402f1
yarn
SAS_SUBSTRATE_WS_URL=wss://kusama-rpc.polkadot.io yarn start
```
and then `curl localhost:8080/block/900` or any other lower number block to get the above errors.

# How to produce expected behavior

To get the expected behavior
```
git checkout master # commit c354be3
yarn
SAS_SUBSTRATE_WS_URL=wss://kusama-rpc.polkadot.io yarn start
```
and then `curl localhost:8080/block/900` or any other valid block number to get a 200 response.

### Other notes

- I get the same errors with wss://cc3-5.kusama.network
- I don't get the error with `curl localhost:8080/block/0`, presumably because it has no extrinsics
- I don't get similar errors with wss://rpc.polkadot.io

**Comment by emostov** [comment]
I narrowed the error for Sidecar down to @polkadot/api@1.30.0-beta.3 - which then leads me to think it has something to do with #2502.

You can checkout Sidecar branch [zeke-test-3 (commit 0503019)](https://github.com/paritytech/substrate-api-sidecar/blob/zeke-test-3/yarn.lock#L501), which uses 1.30.0-beta.3 to produce the issue. And you can also see that it works with 1.30.0-beta.2 by checking out [zeke-beta2 (commit c60009e)](https://github.com/paritytech/substrate-api-sidecar/blob/zeke-beta2/yarn.lock#L501).

I will add though that I tried out an [isolated example of `getBlock` with 1.30.0-beta.6 here](https://github.com/emostov/substrate-tx-seed/blob/1-30-err/src/main.ts) and it works fine.

**Comment by jacogr** [comment]
Thanks for testing the betas. The Kusama know-types were actually non-sensical where it overrode LookupSource. Fixed in #2509
